code/controllers/master_controller.dm

```diff
@@ -38,8 +40,15 @@ var/global/pipe_processing_killed = 0
 		job_master.LoadJobs("config/jobs.txt")
 		log_startup_progress("Job setup complete in [stop_watch(watch)]s.")
 
-	if(!syndicate_code_phrase)		syndicate_code_phrase	= generate_code_phrase()
-	if(!syndicate_code_response)	syndicate_code_response	= generate_code_phrase()
+	if(!syndicate_code_phrase)
+		var/list/new_phrases = generate_code_phrase()
+		syndicate_code_phrase = n_jointext(new_phrases, ", ") + "."

```

Comment:
`n_jointext` is just a `call`able version of the built-in `jointext`, which you should use directly in normal code.